pkg/KubeArmorAnnotation/Makefile

```
@@ -78,9 +78,31 @@ docker-push: ## Push docker image with the manager.
 
 ##@ Deployment
 
-deploy-cert-manager: ## Deploy cert-manager before deploying the admission controller
+deploy-cert-manager: ensure-cmctl ## Deploy cert-manager before deploying the admission controller
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
 	kubectl wait pods --for=condition=ready -n cert-manager -l app.kubernetes.io/instance=cert-manager
+	@echo "Waiting for cert manager to inject CA bundle, this can take up to 5 minutes"
+	$(CMCTL) check api  --wait 300s
+
+CMCTL = $(CURDIR)/bin/cmctl
```

```suggestion
CMCTL = $(CURDIR)/cmctl
```
So that we don't get the error due to parent 'bin' directory not being present.
```
Installing ...
mv: cannot move 'cmctl' to '/home/rudraksh/dev/kubearmor/pkg/KubeArmorAnnotation/bin/cmctl': No such file or directory
make: *** [Makefile:89: ensure-cmctl] Error 1
```

---

If we'll be needing `cmctl` everytime, should we install it on path? :thinking:

---

Good idea, PR updated.